src/Tasks/GenerateApplicationManifest.cs

```diff
@@ -386,7 +386,7 @@ private bool BuildResolvedSettings(ApplicationManifest manifest)
             }
             else if (String.IsNullOrEmpty(manifest.Publisher))
             {
-                string org = Util.GetRegisteredOrganization();
+                string org = NativeMethodsShared.IsWindows ? Util.GetRegisteredOrganization() : string.Empty;
```

Comment:
net5.0+ complains about code you can't hit on other platforms, so we need to "guard" those codepaths. This PR is littered with changes like this. Where I can, I've added guards around the class instead of little spots like this. Some cases are necessary (like this one) because it's a task that could theoretically be run from anywhere.

Reply:
Hmm...ok. It worries me a bit because if it actually does get run on non-windows, this will be very confusingly wrong, right? I'd ideally want something at the top that just fails if not windows, and then you don't have to worry about it as much.

Reply:
Right, but the entry point is `Execute` which can be called via `<Foo` in any `.targets` file which we can't really control without implementing some new mechanism to determine before a task runs that it's windows only.

Reply:
Yeah—I agree that the task could be called on any OS; I was just thinking it would be good to fail fast and obviously. I'm assuming you've tested it, and you get reasonable behavior on mac/linux, but if someone tweaks the task just slightly, it might return something wrong and confusing. If you fail immediately right when they call execute if it isn't windows, it's hard to mess that up.

Reply:
Bump rainersigwald's suggestion

Reply:
Blarg, I had a pending comment here I didn't publish. I'm sold on the idea of applying it to the whole class _and_ erroring in the Execute method.